docs/installation.md

```diff
@@ -89,44 +89,56 @@ For example,
   ```
 ---
 
-# Usage
-## 1. Modify default.yaml (VATZ)
+# 4. Set VATZ configs for your own use
+
+## 4.1. Modify default.yaml (VATZ)
 There's yaml file as config, and the default config must be updated by yourself to set up VATZ service and clients
```

Comment:
There's a YAML file for configuration, and you'll need to update the default settings yourself to properly set up the VATZ service and clients.

Reply:
updated!